README.md

```diff
@@ -42,6 +42,14 @@ compile 'com.roughike:bottom-bar:2.3.1'
 </dependency>
 ```
 
+## ProGuard?
+
+Do add rules below if using ProGuard:
+
+```
+-dontwarn com.roughike.bottombar.**
```

Comment:
don't see any problem with adding this, but, is this an issue in the library, or is something that is not an issue and is just gradle complaining? so that i understand what's the problem we are trying to solve.